MonoGame.Framework/Input/GamePadThumbSticks.cs

```diff
@@ -63,7 +63,7 @@ internal GamePadThumbSticks(Vector2 leftPosition, Vector2 rightPosition, GamePad
             else
                 ApplySquareClamp();
 
-            SetVirtualButtons(leftPosition, rightPosition);
+            SetVirtualButtons(left, right);

```

Comment:
Woops, this is a leftover from my incorrect test/fix that's mentioned in the discussion. It doesn't really matter in practice here, but left/right are the processed values. As in they are set to 0 0 when they fall in the dead zone. IMo it's clearer if I revert this change.